src/registrar/models/very_important_person.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+from .utility.time_stamped_model import TimeStampedModel
+
+
+class VeryImportantPerson(TimeStampedModel):
+
+    """"""
+
+    email = models.EmailField(
+        null=True,
+        blank=True,
+        help_text="Email",
+        db_index=True,
+    )
+
+    user = models.ForeignKey(
+        "registrar.User",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="verifiedby_user",
+    )
+
+    notes = models.TextField(
+        null=True,
+        blank=True,
+        help_text="Notes",
+    )
```

Comment:
(Blocking) All three fields in this file should be required (with user as maybe optional), but especially the notes field. 

There should never be a scenario in which:
1. Email is None. This defeats the purpose of the table.
2. Notes is None. The scenarios in which we want to grant access will _always_ require a justifiable reason to do so, and we should always write that down.
3. User is None. This one may be OK because we have an audit log and other logic you implemented, but if we can get away with making this required I think we should. If its a pain to do so, its not a biggie

Reply:
On 2: I would run that by design and product for confirmation
cc @Katherine-Osos @PaulKuykendall